lib/reporters/console_reporter.js

```diff
@@ -10,7 +10,7 @@ function ConsoleReporter() {
     showColors = false,
     timer = noopTimer,
     jasmineCorePath = null,
-    printDeprecation = function() {},
+    printDeprecation = require('../printDeprecation'),
```

Comment:
We're actually doing this in the previous manner so that no deprecation errors are printed out during specs. This also allows the specs not not have to pass in a custom no-op `printDeprecation` if that particular spec isn't concerned with the deprecated functionality.